images/Makefile

```diff
@@ -16,6 +16,8 @@ include image.mk
 
 PLATFORMS ?= linux_amd64 linux_arm64
 
+IMAGES ?= ceph ceph-toolbox cockroachdb minio
```

Comment:
so we can remove this now since image.mk includes common.mk, where this variable is defined?

Reply:
Gah! I guess this got missed in a rebase.